### DISCUSSION TITLE
Curious if the save and load of the story and scenario is possible to be directly loaded using llama.cpp args

### DISCUSSION THREAD

**Discussion by hgftrdw45ud67is8o89** [LostRuins/koboldcpp#670] [Q&A] [answered]
Is the feature manually implemented by koboldcpp? or is it actually a command flag feature implemented by llama.cpp?

**Comment by LostRuins** [comment, marked as answer]
check out the wiki, under the section for `--preloadstory`

**Reply by LostRuins** [reply]
it is a koboldcpp exclusive.

**Reply by hgftrdw45ud67is8o89** [reply]
Huge thanks! ^_^
